ydb/library/yql/udfs/common/datetime2/datetime_udf.cpp

```diff
@@ -862,8 +869,47 @@ NUdf::TUnboxedValuePod DoAddYears(const NUdf::TUnboxedValuePod& date, i64 years,
     };
 
     // From*
+    
+    template<typename TInput, typename TOutput, TOutput Core(TInput), bool Filter(TInput)>
+    struct TUnaryFixedSizeKernelWithFilter {
```

Comment:
please add suffix FixedUnsafe and ensure that TInput/TOutput are std::is_arithmetic